cdc/owner_operator.go

```diff
@@ -45,7 +45,7 @@ type ddlHandler struct {
 func newDDLHandler(pdCli pd.Client, credential *security.Credential, kvStorage tidbkv.Storage, checkpointTS uint64) *ddlHandler {
 	// The key in DDL kv pair returned from TiKV is already memcompariable encoded,
 	// so we set `needEncode` to false.
-	plr := puller.NewPuller(pdCli, credential, kvStorage, checkpointTS, []regionspan.Span{regionspan.GetDDLSpan(), regionspan.GetAddIndexDDLSpan()}, nil)
+	plr := puller.NewPuller(pdCli, credential, kvStorage, checkpointTS, []regionspan.Span{regionspan.GetDDLSpan(), regionspan.GetAddIndexDDLSpan()}, nil, false)
```

Comment:
This comment can be removed now, or user may think the false means `needEncode=false` by mistake. : )
also recommend adding comments like
```
    plr := puller.NewPuller(
        ...,
        nil,   /* limitter */
        false, /* enableOldValue*/
    )
```